doc/building_windows.md

```diff
@@ -0,0 +1,213 @@
+	* [SDL2_mixer dev libraries for Visual Studio](https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-devel-2.0.4-VC.zip).
+
+3. Extract the `SDL2` and `SDL2_mixer` development libraries to an easy to remember folder (I recommend something like `C:\libs`).
+
+4. Open Microsoft Visual Studio. Select `Clone or check out code`.
+ 
+ 	![](images/building_windows_msvc_1.jpg)
+
+5. In the `Repository location` option, type `https://github.com/bvschaik/julius.git`.
+   In the `Local path` option, select in which folder you wish the source to be downloaded to. This will be your `work folder`:
+
+    ![](images/building_windows_msvc_2.jpg)
+
+    Alternatively, you can obtain the source code without using Visual Studio's built in Git integration. Check [obtaining the source files for Julius](#obtaining-the-source-files-for-julius) for details.
+
+6. The first time you open the project, Visual Studio will throw an error complaining it can't find SDL:
+
+	![](images/building_windows_msvc_3.jpg)
+
+7. You will need to tell Visual Studio where the SDL libraries you downloaded can be found. To do that, click up top where it says `x64-Debug` and select `Manage Configurations`.
+
+8. On the new screen that appears, scroll down to `CMake Command Arguments`.
+
+9. Paste the location of your library paths. If you used the above example paths, then paste the following code:
+
+	`-DCMAKE_PREFIX_PATH=C:\\libs\\SDL2-2.0.10;C:\\libs\\SDL2_mixer-2.0.4`
+
+	Notice that you must always include `-DCMAKE_PREFIX_PATH=` at the start and that you must use double backward slashes.
+
+	![](images/building_windows_msvc_4.gif)
+
+10. Press `CTRL` + `S` to save the configuration. Visual Studio will attempt to reconfigure everything, and this time it should detect the libraries and work.
+
+11. Pick `Select Startup Item` up top and select `julius.exe`.
+
+	![](images/building_windows_msvc_5.gif)
+
+12. Go to `Build -> Build All`. The game should compile without errors.
+
+13. After the game compiles, it is stored in a folder inside your `work folder`. You can access it from `work folder\out\build\x64-Debug`.
+
+14. If you simply execute the game right now, it will complain about missing DLL's. You need to copy the `SDL2` and `SDL2_mixer` DLL's to the same directory as the executable.
+
+	Both libraries are available inside your library directory. Using the example library location, the files should be in the following location:
+
+	* `SDL2` - `C:\libs\SDL2-2.0.10\lib\x64`
+	* `SDL2_mixer` - `C:\libs\SDL2_mixer-2.0.4\lib\x64`
+
+**Success!** You should now be able to run the game.
+
+
+## B. Using CMake and MinGW via command line
+
+Building Julius by command line is useful if you don't wish to install a very large IDE such as Visual Studio. However, it still requires that you install some programs, such as CMake, MinGW and optionally Git.
+
+
+### Installing the prerequesite programs
+
+#### Installing MinGW
```

Comment:
Why MinGW and not MinGW-w64?

Reply:
Because travis builds for 32 bit, so I kept that for consistency (I kept msvc at 64 bits because it's the default)

Reply:
MinGW-w64 can also do 32-bits builds. I thought I moved to MinGW-w64 for the AppVeyor builds, but that was on my branch for creating stacktraces upon crashes.

Reply:
There is a MinGW-32 and a 64 bit appveyor configuration. I'll have to check whether the installer I provide has the 64 bit compiler. The default installer for mingw seems to be difficult to install for a regular user, so I used that one.

Reply:
Ok so now that I read about mingw-w64 (I've always used Visual Studio) I'm feeling a bit stupid. MinGW-w64 is a fork of mingw, not the 64 bit version!

Yeah I'll change to MinGW-w64, it's easier to install as well!